introducing-metrics-in-innersource.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Introducing Metrics in the InnerSource Initiative
+
+# Context
+
+An organization is applying InnerSource. This needs to check the
+current status and if some of the expected outcomes are starting
+to work.
+
+
+# Problem
+
+This organization may already have qualitative feedback from the
+involved teams, but desire more objective information focused
+on development activities.
+
+The organization does not really know where to start measuring
+things or what are the key parameters to measure.
+
+
+# Forces
+
+People do not like to be tracked or measured.
+
+There is not a proper monitoring infrastructure for the software
+development process and this is hard to build or to get funding
+for this.
+
+There is not a culture of software development metrics.
+
+Metrics are usually misunderstood if people have not received any
+training on those.
+
+
+# Solution
+
+Bring developers, middle managers and C-level to have a discussion
+about metrics.
+
+Let developers and middle managers know that these metrics or KPIs
```

Comment:
@gruetter 

A "neutral third party" might invite bureaucracy, unless its scope is limited to statistical validity. I believe that organizations should create relevant definitions in the following order: 

1. Aim
1. Business objectives
1. Metrics
1. KPIs

I'd love to hear alternative approaches, though!

Reply:
I'd go for a third party as playing a neutral role in the sense of understanding the needs from C-level, middle management, developers, and others and then implement those in a way understandable by everyone.

This would help when avoiding suspicious behaviours at any level.

@gregswindle having an approach as the one you mention is great. There is a clear need for defining a strategy and method to work on metrics. Indeed I'd go for a couple of extra patterns with this respect that we could reference from here. Does it make sense?